### ISSUE TITLE
No operations defined in spec on SwaggerUI when I run the "serverless offline start" command

### ISSUE THREAD

**Issue by mkumail1** [SamWSoftware/serverless-auto-swagger#36]
I did everything as instructed in this video: https://www.youtube.com/watch?v=vkTIM9MQ5Wc, but for some reason, the 'serverless offline start' command after running on the localhost of the browser shows me a message 'no operations defined in spec' upon running the offline command. Can anyone help? I am using esbuild instead of webpack. 

![image](https://user-images.githubusercontent.com/45015600/153854834-af1f50c6-64c4-431a-bc09-34f95723d211.png)

![image](https://user-images.githubusercontent.com/45015600/153854025-f5660709-e909-4b32-adcc-757bb882abdc.png)

Although swagger.json is returning the right config: 
![image](https://user-images.githubusercontent.com/45015600/153852170-bb6a4636-0dd3-447c-95dc-dd19ccafb4c8.png)

I am using:
"@serverless/typescript": "^3.0.0"
"esbuild": "^0.14.11",
"typescript": "^4.1.3"
"serverless-offline": "^8.4.0"
"node": ">=14.15.0"

**Comment by hukus** [comment]
You need to use lowercase letters for your operation names. It doesn't work with properties named with capital letters.

So change `DELETE` into `delete`

**Comment by mkumail1** [comment]
So I solved this issue by generating my own Swagger JSON from Lambda API Gateway. And manually configuring in autoswagger yml config. 

![image](https://user-images.githubusercontent.com/45015600/154137080-789b5a57-a054-4b6e-a504-4d3a567946a1.png)

Then I was landed on another issue which was that the swagger lambda API was hosting on another URL from the rest of my function URLs. This resulted in an error in the production because AWS refrains from cross-communicating APIs across different URLs.

something like this: 
![image](https://user-images.githubusercontent.com/45015600/154137892-a804a8f1-efa7-45ad-a0bc-1c41efcd002f.png)

Then after hours of debugging, I solved the issue above by adding these lines in my swagger.config inside the request scope:
![image](https://user-images.githubusercontent.com/45015600/154138206-fe44ee7a-03b2-4ef9-bb53-318d9b86a433.png)
